incuna-sass/mixins/_grid.sass

```diff
@@ -0,0 +1,37 @@
+// Mixin to add a grid background to elements
+
+// Can be used to create a grid lines on their own or with a solid color background
+// Direction can be set to create just horizontal or just vertical lines, or left
+// as 'both' to create a grid
+
+// Example usage:
+// .element
+//     @include grid(4px, blue)
+// will create a transparent background with a grid that has 4px squares and 1px opaque
+// blue grid lines in both directions
+
+// .element
+//     @include grid(10px, red, 0.4, orange)
+// will create an orange background with a grid that has 10px squares and 1px red lines
+// at 40% opacity
+
+// .element
+//     @include grid(20px, green, 1, transparent, 5px, horizontal)
+// will create a transparent background with 5px opaque green lines going horizontally
+// across the page 20px apart
+
+@mixin grid($grid-size, $grid-color, $grid-opacity: 1, $bg-color: transparent, $line-thickness: 1px, $direction: both)
```

Comment:
This just seems excessively abstract, like I don't really understand what's going on.

Possibly if I did some research on sass maps I might get it, but it's probably something we can talk about in general, not something that needs to be implemented on this particular thing